src/render_graph.rs

```diff
@@ -159,6 +159,19 @@ impl Node for PointCloudNode {
             tracked_pass.set_camera_viewport(viewport);
         }
         tracked_pass.set_render_pipeline(eye_dome_pipeline);
+
+        let edl_strength: f32 = if view.projection.z_axis.w == -1.0 {
```

Comment:
It's a shame the extracted components don't have any other way of knowing the original type of projection. It would be cleaner to check the `Projection` component on the camera entity to know if it's ortho or perspective.

Also you can omit the type annotation.

Reply:
Yeah, I agree. It never hurts to include the type annotation tho.